vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -1568,4 +1584,14 @@ private void removeLazyOpenRegistration() {
         }
     }
 
+    private void onDataChange(SerializablePredicate<T> filter,
```

Comment:
Data change ?  In this context I see data as something else than filtering & sorting

Reply:
Renamed here and on the flow side also.